template/next-ts-chakra-ui/src/components/Provider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
[not sure how to add file-level comment]. Is `Provider` too general a name for this? Feels like we may as well have `Component.tsx`

Reply:
I'm not too sure what you mean by this. Are you saying you think it should be more specific, like `GlobalProvider` or `AppProvider`?

Reply:
Thats what I'm getting at yeah - it's basically a ThemeProvider or StyleProvider at the moment right? 
Is the pattern that it would begin to provide multiple things? Or we would have
```
<Multiple>
  <LevelsOfProviders>
    <WrappingThe>
       <PageComponent />
```

Reply:
The idea was that the one component would provide everything, since it needs to be used in 3 places (`_app.tsx`, Storybook and Jest) I'd rather not have to change all 3 places because I added Redux or what have you.

Reply:
I've renamed it to `AppProvider` 👍